.github/bors.toml

```diff
@@ -0,0 +1,3 @@
+status = [ "Bors" ]
```

Comment:
Could you add the usual copyright and license, please?

Reply:
Will do, thanks for noticing!

Reply:
Done. I made the email "contact@stellar-group.org". I see the documentation repo has commits from "StellarBot" with "stellar@cct.lsu.edu". Do you remember if I'd need to actually use the latter?

Reply:
> Done. I made the email "[contact@stellar-group.org](mailto:contact@stellar-group.org)". I see the documentation repo has commits from "StellarBot" with "[stellar@cct.lsu.edu](mailto:stellar@cct.lsu.edu)". Do you remember if I'd need to actually use the latter?

I wouldn't even know where this email is configured :/

Reply:
I think it's inconsequential: https://github.com/STEllAR-GROUP/hpx/blob/8a6a72b023e01ad4a884835f2d7e48fcfaf7e988/.circleci/config.yml#L75. I'll change that one to use "contact@stellar-group.org" as well.